src/main.js

```diff
@@ -10,6 +10,7 @@ import store from './store.js'
 import { useUserStore } from '@/stores/index.js'
 import { getOperatorParcelles } from './cartobio-api.js'
 import App from './App.vue'
+import version from "../package.json"
```

Comment:
```suggestion
import { version } from "../package.json"
```

Ça ne serait pas plutôt ça ?